interview-questions.md

```diff
@@ -8,33 +8,33 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: The main difference between a parameter and an argument is that the parameter is  placeholder for data that is passed into a function, while an argument is the actual data that is passed through into the function.
 
-Researched answer:
+Researched answer: When referring to JavaScript functions, parameters and arguments are used interchangeably, however this is not the case, and there is a very suble difference. Parameters are variables listed within the function definition, whereas arguements are values that are PASSED to the function when invoked.
```

Comment:
Because the first answer was closely worded to an article on the following website https://codeburst.io/parameters-arguments-in-javascript-eb1d8bd0ef04, I am requesting that you place this answer into your own words.